src/ScriptBuilderTask/ScriptBuilderTask.csproj

```diff
@@ -1,23 +1,24 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFrameworks>net452;netstandard1.5</TargetFrameworks>
     <AssemblyName>NServiceBus.Persistence.Sql.ScriptBuilderTask</AssemblyName>
     <RootNamespace>NServiceBus.Persistence.Sql</RootNamespace>
-    <AssemblyTitle>NServiceBus.Persistence.Sql.ScriptBuilderTask</AssemblyTitle>
-    <Product>NServiceBus.Persistence.Sql.ScriptBuilderTask</Product>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>$(SolutionDir)NServiceBus.snk</AssemblyOriginatorKeyFile>
     <DebugType>full</DebugType>
     <OutputPath>..\..\binaries\</OutputPath>
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
```

Comment:
I had to look this up - it silences "Missing XML comment for publicly visible type or member" for ScriptBuilderTask.

Reply:
I had to add that because using Particular.Packaging will cause a project to create an xml documentation file, and this project wasn't doing that before, so it started spitting out warnings.